.gitignore

```diff
@@ -12,8 +12,14 @@ wp-content/uploads
 
 # these are customized by the setup
 composer.json
+!setup/composer.json
 deploy-exclude-list.txt
+!setup/deploy-exclude-list.txt
 deploy.sh
+!setup/deploy.sh
 gulpfile.js
+!setup/gulpfile.js
 package.json
-
+!setup/package.json
+backup-db.sh
+!setup/backup-db.sh
```

Comment:
It is my understanding that the bang prefix for gitignore means that you will not ignore those files. I was expecting to see that you had set `setup/` as a gitignore and were making all those other files exceptions. ??